### PR TITLE
Fixing typo in cucumber scenario

### PR DESCRIPTION
**Pull request by brian-dlee**

### CODE REVIEW

**Review by matthandlersux — APPROVED** [review]
ship it, and thanks for fixing :)

@ChrisMM fyi in case this needs to be redeployed to originate.com after merging. is that automatic?

**Comment by brian-dlee** [comment]
@matthandlersux I just started with Originate on Monday, and I don't have write access to merge the pull request. Would you be able to assist with that?

**Comment by ChrisMM** [comment]
@matthandlersux Unfortunately, it's not automatic. I'll get it updated. Thanks!

**Pull request merged**
